editor/editor_layouts_dialog.cpp

```diff
@@ -95,8 +95,7 @@ void EditorLayoutsDialog::_post_popup() {
 		return;
 	}
 
-	List<String> layouts;
-	config.ptr()->get_sections(&layouts);
+	Vector<String> layouts = config.ptr()->get_sections();
```

Comment:
Not really related but
```suggestion
	Vector<String> layouts = config->get_sections();
```